src/devices/LidarLiteV3/LidarLiteV3.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <EnableDefaultItems>false</EnableDefaultItems>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="$(MainLibraryPath)System.Device.Gpio.csproj">
+      <AdditionalProperties>$(AdditionalProperties);RuntimeIdentifier=linux</AdditionalProperties>
+    </ProjectReference>
+    <PackageReference Include="System.Memory" Version="4.5.3" />
+    <Compile Include="*.cs" />
+    <None Include="README.md" />
+    <Compile Include="..\Common\System\Device\DelayHelper.cs" Link="DelayHelper.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Reference Include="System.Numerics.Vectors" />
+  </ItemGroup>
+
+</Project>
```

Comment:
can't put comment there. Please fill the `src/devices/LidarLiteV3/README.md` with at minimum a reasonable title (see other devices) and a tiny description. README.md is used for generating the [device listing](https://github.com/dotnet/iot/blob/master/src/devices/README.md)

Reply:
Left a more detailed README in the sample folder, and added a summary README for the device